docs/usage.md

```diff
@@ -5,90 +5,225 @@
+## OT - Add Sample Assets
 
-1. **OT - Add Sample Assets**
+This scenario generates 87 IT/OT assets as per the Purdue model based on various criticality levels, types, and other asset categorizations as sample data.
 
-    - Browse to `Simulations` > `OT - Add Sample Assets` scenario and click **Simulate Scenario**.
+To run this scenario and generate the assets, perform the following steps:
 
-    - This scenario adds 86 different sample OT Assets (different Levels, Types etc.) for testing dashboards, use case playbooks, reports and other actions.<br>
+1. Under the FortiSOAR&trade; menu select to open **Simulations**.
 
-2. **OT - Add Sample Alerts**
+2. Select the scenario **OT - Add Sample Assets**.
 
-    - Browse to `Simulations` > `OT - Add Sample Alerts` scenario and click **Simulate Scenario**.
+3. Click **Simulate Scenario**.
 
-    - This scenario adds 12 well populated sample OT Alerts for testing dashboards, use case playbooks, reports and other actions.<br>
+4. To view the added sample assets, under the FortiSOAR&trade; menu select to open **Resources** > **Assets**.
 
-3. **OT - Stuxnet Attack Scenario**
+![](./res/added-sample-assets.png)
 
-    - Make sure the global variable **Demo mode** is set to `true`. 
+## OT - Add Sample Alerts
 
-    - Browse to `Simulations` > `OT - Stuxnet Attack Scenario` scenario and click **Simulate Scenario**.
+This scenario generates 12 well-populated alerts associated with assets, generated in the previous scenario, as sample data.
 
-    - Four Alerts get created each at an interval of 2 seconds:
-        - Stuxnet peer to peer communication attempt
-        - Anomalous communication between two Windows XP machines
-        - Communication between public and private networks
-        - STEP7 configuration download command
+To run this scenario and generate the alerts, perform the following steps:
 
-    - If an asset does not already exist, it is created and added to the record with the following hostname:
-        - PLC S7-400-324
-        - Dell - 5490-42E6
-        - ThinkCentre M910s-1181
-        - HP - PB7181
+1. Under the FortiSOAR&trade; menu select to open **Simulations**.
 
-    - Each alert get associated with assets based on the Source and Destination IP.
+2. Select the scenario **OT - Add Sample Alerts**.
 
-    - **Group**, **Tactics**, **Technique**, **Sub-Technique**, and **Software** get linked to alert under `Correlations` tab of Alert.
-    
-    - **Mitigation** get correlated to alert `Recommended ATT&CK Mitigations` tab.
+3. Click **Simulate Scenario**.
 
-    - Once the alert has been enriched, open the `Stuxnet peer to peer communication attempt` alert and look for a similar alert in the Workspace `Recommendations` tab. 
+4. To view the added sample alerts, under the FortiSOAR&trade; menu select to open **Incident Response** > **Alerts**.

```

Comment:
Please check the phrase is it making sense

Reply:
Fixed